invenio_records_rest/config.py

```diff
@@ -43,6 +44,7 @@ def _(x):
         pid_minter='recid',
         pid_fetcher='recid',
         search_class=RecordsSearch,
+        indexer_class=RecordIndexer,
```

Comment:
I think likely this should be an object instance and not a class. This will allow to customize the indexer with the parameters needed (e.g. specific queue names etc)

Reply:
Right, it's changed to indexer and we had to pass `indexer=lambda: RecordIndexer()` because of this line in conftest: `RECORDS_REST_ENDPOINTS=copy.deepcopy(config.RECORDS_REST_ENDPOINTS)` which will complain about the missing application context.